.github/workflows/tox.yaml

```
@@ -37,3 +37,24 @@ jobs:
         run: pip install tox
       - name: Run Tox
         run: tox
+
+  integration-test:
+    name: Integration test with VMWare
+    runs-on: self-hosted
+    timeout-minutes: 360
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Setup operator environment
+        uses: charmed-kubernetes/actions-operator@main
+        with:
+          provider: vsphere
+          credentials-yaml: ${{ secrets.CREDENTIALS_YAML }}
+          clouds-yaml: ${{ secrets.CLOUDS_YAML }}
+          bootstrap-options: "--model-default datastore=vsanDatastore --model-default primary-network=VLAN_2763"
```

could i offer this -- it will speed up the integration tests run by using an image for the controller that's already got the VM better configured for juju in focal

```suggestion
          clouds-yaml: ${{ secrets.CLOUDS_YAML }}
          bootstrap-constraints: "arch=amd64 cores=2 mem=4G"
          bootstrap-options: "${{ secrets.FOCAL_BOOTSTRAP_OPTIONS }} --model-default datastore=vsanDatastore --model-default primary-network=VLAN_2763"         
```

---

Thank you, will add this and the fix for the linting errors in a new commit.